src/Query/Admin/UserQuery/DbalUserQuery.php

```diff
@@ -73,7 +71,7 @@ private function hydrateUser(array $data): User
             $data['id'],
             $data['email'],
             $data['status'],
-            json_decode($data['roles'])
+            json_decode($data['roles'], null, 512, JSON_THROW_ON_ERROR)
```

Comment:
Change in behaviour (different exception being thrown now if I'm correct)?

But I believe it's fine here.